### DISCUSSION TITLE
Slow web UI

### DISCUSSION THREAD

**Discussion by apastuszak** [rern/rAudio#337] [General]
When I use the web UI, the UI gets unresponsive after a while. So, I'll switch to another tab and come back to Rune Audio a song or two later, and if I click on a control, it will take 5 seconds to respond. Once it does, then the Web GUI is very responsive.

Is this normal?

I'm using a Raspberry Pi 3B+. If I got a 4 GB Pi 4, would this problem go away?

**Comment by apastuszak** [comment]
After some more testing, this appears to only be an issue in Firefox.  I tried it in a Chromium based browser (MS Edge) and it worked just fine. Safari on Mac seems to work fine.

Firefox is my primary browser, so I would like to see this fixed.  But I am not sure if this is a Firefox or a Raudio-1 issue.

**Reply by AndyFx12** [reply]
What version of Firefox do you use? Latest Firefox works fine on my system. 
Andy

**Reply by apastuszak** [reply]
I did more testing last night. Seems to be Firefox on Mac Intel specifically.  When I tried this on Firefox Linux last night, it worked no problem.
